runner/lib/main.js

```diff
@@ -501,6 +520,57 @@ const main = async (progName, rawArgs, powers) => {
       stageConfig: config,
     });
 
+    /**
+     * @type {Task}
+     *
+     * This task will always run in three stages
```

Comment:
```suggestion
     * This task should always run in three stages
```

It doesn't seem to hurt if more stages are run, we just immediately complete the stage (after chain catch up of course)

Reply:
Done